change/@fluentui-react-tree-fc112f7a-3c5a-4302-bca4-4c367d5266c5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32232&commitId=838bd99189ad19f01bd77d02853e23d2b5751e84&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Tree</b> 32 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.appearance - Dark Mode.Subtle alpha tree hover.chromium.png|155| Changed|
|Tree.appearance - Dark Mode.Subtle alpha tree mousedown.chromium.png|155| Changed|
|Tree.appearance - Dark Mode.Subtle tree hover.chromium.png|3334| Changed|
|Tree.appearance - High Contrast.Subtle alpha tree hover.chromium.png|77968| Changed|
|Tree.appearance - Dark Mode.Subtle tree mousedown.chromium.png|3334| Changed|
|Tree.actions.chromium.png|189| Changed|
|Tree.actions - RTL.chromium.png|188| Changed|
|Tree.actions - Dark Mode.chromium.png|172| Changed|
|Tree.appearance - Dark Mode.Transparent tree mousedown.chromium.png|173| Changed|
|Tree.actions - High Contrast.chromium.png|254| Changed|
|Tree.appearance - High Contrast.Transparent tree hover.chromium.png|77670| Changed|
|Tree.appearance - Dark Mode.Transparent tree hover.chromium.png|173| Changed|
|Tree.appearance - High Contrast.Subtle tree hover.chromium.png|38854| Changed|
|Tree.appearance - High Contrast.Subtle alpha tree mousedown.chromium.png|77968| Changed|
|Tree.appearance - High Contrast.Transparent tree mousedown.chromium.png|77670| Changed|
|Tree.appearance - High Contrast.Subtle tree mousedown.chromium.png|38854| Changed|
|Tree.appearance - RTL.Subtle alpha tree hover.chromium.png|7927| Changed|
|Tree.appearance - RTL.Subtle alpha tree mousedown.chromium.png|7927| Changed|
|Tree.appearance - RTL.Transparent tree mousedown.chromium.png|7912| Changed|
|Tree.appearance - RTL.Transparent tree hover.chromium.png|7912| Changed|
|Tree.appearance.Subtle alpha tree mousedown.chromium.png|8432| Changed|
|Tree.layout - Dark Mode.chromium.png|123| Changed|
|Tree.layout - High Contrast.chromium.png|401| Changed|
|Tree.appearance.Transparent tree hover.chromium.png|8411| Changed|
|Tree.layout - RTL.chromium.png|286| Changed|
|Tree.appearance.Subtle alpha tree hover.chromium.png|8432| Changed|
|Tree.appearance.Transparent tree mousedown.chromium.png|8411| Changed|
|Tree.persona - RTL.chromium.png|1059| Changed|
|Tree.layout.chromium.png|286| Changed|
|Tree.persona - Dark Mode.chromium.png|734| Changed|
|Tree.persona - High Contrast.chromium.png|1517| Changed|
|Tree.persona.chromium.png|1059| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>